filter_noise_messages.py

```diff
@@ -0,0 +1,35 @@
+import json
+import argparse
+
+from core_data_modules.cleaners import somali
+from core_data_modules.logging import Logger
+from core_data_modules.util import IOUtils
+
+log = Logger(__name__)
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Filter out noise messages from Coda Files",
+                                     # Support \n and long lines
+                                     formatter_class=argparse.RawTextHelpFormatter)
+    parser.add_argument("coded_dir_path", metavar="coded-dir-path",
```

Comment:
Doing it in place is a bit scary. Consider taking separate input and output directories

Reply:
updated thanks